### PR TITLE
Fix pr comments

### PR DESCRIPTION
**Pull request by zbud-msft**
**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: src/sonic-eventd/rsyslog_plugin/rsyslog_plugin.cpp [hunk 1]
```diff
@@ -25,50 +25,77 @@ bool RsyslogPlugin::onMessage(string msg, lua_State* luaState) {
     }
 }
 
+void parseParams(vector<string> params, vector<EventParam>& eventParams) {
+    for(long unsigned int i = 0; i < params.size(); i++) {
+        if(params[i].empty()) { continue; }
```
Comment (renukamanavalan) [inline src/sonic-eventd/rsyslog_plugin/rsyslog_plugin.cpp]: Is this even possible?

There may be no params. But if there is one, it at the least has the name. If not it is a bug in the i/p file.

File: src/sonic-eventd/rsyslog_plugin/rsyslog_plugin.cpp [hunk 2]
```diff
@@ -25,50 +25,77 @@ bool RsyslogPlugin::onMessage(string msg, lua_State* luaState) {
...
+        EventParam ep = EventParam();
+        auto delimPos = params[i].find(':');
+        if(delimPos == string::npos) { // no lua code
+            ep.paramName = params[i];
+        } else {
+            ep.paramName = params[i].substr(0, delimPos);
+	    ep.luaCode = params[i].substr(delimPos + 1);
+            if(ep.luaCode.empty()) {
+                SWSS_LOG_INFO("Lua code missing after :");
```
Comment (renukamanavalan) [inline src/sonic-eventd/rsyslog_plugin/rsyslog_plugin.cpp]: This can be an ERROR message

**Review by renukamanavalan — DISMISSED** [review]
Minor comments. Merge upon update.

File: src/sonic-eventd/rsyslog_plugin/rsyslog_plugin.cpp (hunk 1 above)
Comment (zbud-msft) [reply src/sonic-eventd/rsyslog_plugin/rsyslog_plugin.cpp]: This is just taking care of the edge case where in the regex file

params: [ '', '', '', ''] 

No need to do any type of parsing, just included for edge case

Comment (renukamanavalan) [reply src/sonic-eventd/rsyslog_plugin/rsyslog_plugin.cpp]: Let us report error at least

**Review by renukamanavalan — DISMISSED** [review]

File: src/sonic-eventd/rsyslog_plugin/rsyslog_plugin.cpp (hunk 1 above)
Comment (zbud-msft) [reply src/sonic-eventd/rsyslog_plugin/rsyslog_plugin.cpp]: Done

File: src/sonic-eventd/rsyslog_plugin/rsyslog_plugin.cpp (hunk 2 above)
Comment (zbud-msft) [reply src/sonic-eventd/rsyslog_plugin/rsyslog_plugin.cpp]: Done

**Review by renukamanavalan — APPROVED** [review]

**Pull request merged**